### PR TITLE
Adapt Court to multisortition

### PR DESCRIPTION
**Pull request by bingen**
Remove queues
Add batches

### CODE REVIEW

File: contracts/Court.sol [hunk 1]
```diff
@@ -514,34 +481,35 @@ contract Court is ERC900, ApproveAndCallFallBack {
...
+        while (addedJurors < round.jurorNumber) {
+            (uint256[] memory jurorKeys, uint256[] memory stakes) = _treeSearch(draftTerm.randomness, _disputeId, round.nextJurorToDraft, round.jurorNumber, addedJurors);
+            for (uint256 i = 0; i < jurorKeys.length; i++) {
+                address juror = jurorsByTreeId[jurorKeys[i]];
+
+                // Account storage jurorAccount = accounts[juror]; // Hitting stack too deep
+                uint256 newAtStake = accounts[juror].atStakeTokens + _pct4(jurorMinStake, config.penaltyPct); // maxPenalty
+                if (stakes[i] >= newAtStake) {
+                    accounts[juror].atStakeTokens += newAtStake;
+                    addedJurors++;
```
Comment (bingen) [inline contracts/Court.sol]: This should come later, fixed in commit 42b5327

File: contracts/Court.sol [hunk 2]
```diff
@@ -514,34 +481,35 @@ contract Court is ERC900, ApproveAndCallFallBack {
...
         }
+        round.nextJurorToDraft += uint32(addedJurors);
 
-        dispute.state = DisputeState.Adjudicating;
+        _payFees(config.feeToken, msg.sender, config.draftFee * round.jurorNumber, config.governanceFeeShare);
 
-        _payFees(config.feeToken, msg.sender, config.draftFee * jurorNumber, config.governanceFeeShare);
-
-        emit DisputeStateChanged(_disputeId, dispute.state);
+        if (addedJurors == round.jurorNumber) {
```
Comment (bingen) [inline contracts/Court.sol]: This should check `round.nextJurorToDraft`

File: contracts/Court.sol [hunk 3]
```diff
@@ -514,34 +481,35 @@ contract Court is ERC900, ApproveAndCallFallBack {
...
-                skippedJurors++;
-                i--;
-                continue;
+        // TODO: stack too deep
+        //uint256 jurorNumber = round.jurorNumber;
+        //uint256 nextJurorToDraft = round.nextJurorToDraft;
+        round.votes.length = round.jurorNumber;
+
+        uint256 addedJurors = 0;
+        while (addedJurors < round.jurorNumber) {
```
Comment (bingen) [inline contracts/Court.sol]: This condition is wrong. Refactoring this.

**Comment by facuspagnuolo** [comment]
Closing this PR since `development` branch was already updated with these commits manually

**Pull request closed without merging**